static/src/javascripts/projects/commercial/modules/dfp/prepare-sonobi-tag.js

```diff
@@ -31,19 +31,19 @@ const catchPolyfillErrors = () => {
 };
 
 const setupSonobi: () => Promise<void> = once(() => {
```

Comment:
@rich-nguyen It's OK for this method to return `undefined` as it is only used by `Promise.all`, which can handle value that are not promises. Although it might be worth returning a `Promise.resolve()` anyway so that we at least conform to the type.